test/e2e/tests/connections/multiple-standard-connections.spec.ts

```diff
@@ -0,0 +1,315 @@
+  sessionProperties: {},
+  requiredScopes: {},
+  optionalScopes: {
+    [SOLANA_CAIP_CHAIN_ID]: {
+      accounts: [SOLANA_ACCOUNT_ONE],
+    },
+  },
+};
+
+/**
+ * Helper to open a permissions page for a specific app hostname
+ *
+ * @param driver - The driver to use.
+ * @param hostname - The hostname to get the permissions page for.
+ * @returns The permissions page for the given host.
+ */
+async function getPermissionsPageForHost(driver: Driver, hostname: string) {
+  const homepage = new Homepage(driver);
+  await homepage.headerNavbar.openPermissionsPage();
+  const permissionListPage = new PermissionListPage(driver);
+  await permissionListPage.check_pageIsLoaded();
+  await permissionListPage.openPermissionPageForSite(hostname);
+  const sitePermissionPage = new SitePermissionPage(driver);
+  await sitePermissionPage.check_pageIsLoaded(hostname);
+  return sitePermissionPage;
+}
+
+/**
+ * Helper to get a request permissions request object with a caveat.
+ *
+ * @param caveat - The caveat to add to the request permissions request object.
+ * @returns The request permissions request object with the caveat.
+ */
+function getRequestPermissionsRequestObject(caveat: object = {}): string {
+  return JSON.stringify({
+    jsonrpc: '2.0',
+    method: 'wallet_requestPermissions',
+    params: [{ eth_accounts: caveat }],
+  });
+}
+
+describe('Multiple Standard Dapp Connections', function () {
+  it('should default to existing permitted account when wallet_requestPermissions is called again with no accounts specified', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withKeyringControllerAdditionalAccountVault()
+          .withPreferencesControllerAdditionalAccountIdentities()
+          .withAccountsControllerAdditionalAccountIdentities()
+          .withPermissionControllerConnectedToTestDapp({
+            account: EVM_ACCOUNT_TWO,
+          })
+          .build(),
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver }) => {
+        await loginWithoutBalanceValidation(driver);
+        const testDapp = new TestDapp(driver);
+        const connectAccountConfirmation = new ConnectAccountConfirmation(
+          driver,
+        );
+
+        await testDapp.openTestDappPage();
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
+
+        await testDapp.check_pageIsLoaded();
+
+        await testDapp.check_connectedAccounts(EVM_ACCOUNT_TWO);
+
+        const requestPermissionsWithoutAccounts =
+          getRequestPermissionsRequestObject();
+
+        await driver.executeScript(
+          `window.ethereum.request(${requestPermissionsWithoutAccounts})`,
+        );
+
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+
+        await connectAccountConfirmation.check_pageIsLoaded();
+
+        await connectAccountConfirmation.check_isAccountDisplayed(
+          EVM_ACCOUNT_TWO,
```

Comment:
is checking if account is displayed the same as making sure its checkbox is selected?